ArmPkg/Library/PlatformBootManagerLib/PlatformBm.c

```diff
@@ -1222,7 +1222,8 @@ PlatformBootManagerProcessBootCompletion (
 }
 
 /**
-  OnDemandConInCOnnect
+  OnDemandConInConnect

```

Comment:
This commit is just changing the function comment. It can simply be squashed into the first.

Reply:
Just fixed that. Thanks.

Reply:
I see your PR description says it will squash, but the PR title says `[Rebase & FF]`. You really need to Rebase & FF though to preserve the author info in the original commit. The second I think is fine to add without any additional author info, it's just a trivial single character correction.